.github/workflows/test-add-replica.yaml

```diff
@@ -5,21 +5,21 @@ on:
   workflow_dispatch:
     inputs:
       image:
-        description: 'GCP image for test cluster'
+        description: "GCP image for test cluster"
```

Comment:
Did we converge on a standard for quoting strings in YAML? I thought it was single quotes?